variables.tf

```diff
@@ -93,6 +93,12 @@ variable "replication_group_description" {
   type        = string
 }
 
+variable "route53_zone_id" {
+  default     = ""
```

Comment:
@smiller171 has been encouraging me to use null more. I don't think he's wrong.

Reply:
let me test that, I want to see how it behaves in conditionals

Reply:
In most cases when you use null it omits that parameter entirely.